kured-ds.yaml

```diff
@@ -48,8 +53,13 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+          volumeMounts:
+            - mountPath: /sentinel
```

Comment:
Are we using `/sentinel` as the path inside the container so that we still have access to the container-specific `/var/run` directory? If we don't care about that, using a different name like `sentinel` as the mount path inside the container could confuse some folks.

I see that you and @rptaylor have had a conversation around this, so if you feel O.K. about changing this going forward, I'm lgtm as-is.

The obvious gotcha is that existing, static daemonset files that people are using won't work after we make this change (`/var/run/reboot-required` will point to the local container and not the underlying host filesystem), so they will need to update any of those files to accepted the changes we've added to our `kured-ds.yaml` reference here.

Reply:
The path inside the container is arbitrary, since the hostpath volume can be mounted anywhere. The helm template takes care of mounting volumes in the right place and giving the right path arg to kured in a way that is transparent to the user. The user only needs to specify the location of the sentinel on the host.

I'm not sure I follow. The kured-ds.yml file uses a hostPath to mount /var/run/reboot-required to /sentinel/reboot-required and tells kured to use that location, so there should be no change in behaviour for users.

Reply:
I see your point @jackfrancis. Not all users which use the static manifest may update the whole manifest all time, but instead only update the tag. We will break those users, but I think it should be enough to mention this clearly in the changelog.

Reply:
It looks like this MR has been approved and could be merged? Or is there more to be discussed/agreed here first? @jackfrancis @ckotzbauer 
Thanks!